CityOfDetroit/bloom
# feat: default what to expect text and additional what to expect field

## Issue

- Closes #1054

## Description

Add a new `whatToExpect` text default, and the ability to add a read more section to the what to expect text which lives in the new `whatToExpectAdditionalText` field.

## How Can This Be Tested/Reviewed?

Ensure the default is set on new listings and that existing listings all have their what to expect and additional information fields set to the defaults.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
